### ISSUE TITLE
Consider RawVoice's pre-existing namespace

### ISSUE THREAD

**Issue by theDanielJLewis** [syndicated-media/sn-spec#52]
In discussions of what could be built on top of RSS, I suggest that we also consider the [RSS namespace RawVoice has been building for years](http://www.rawvoice.com/services/tools-and-resources/rawvoice-rss-2-0-module-xmlns-namespace-rss2/).

Their namespace includes:

- rating
- live embed (in multiple formats) and live URL
- location
- frequency
- poster image
- HD video tag
- metamarks (replacement for M4A chapters)
- subscription links
- donation link

**Comment by kookster** [comment]
I think many of those are topics in specific issues, like the last one, donations #14 

Would it be worthwhile to put in each of them as a ticket if there isn't one already, or add a reference to the RV spec in the existing, similar tickets? It seems like this would be a good way to consider their thoughtful and informed additions to RSS, what do you think @theDanielJLewis ?

**Comment by theDanielJLewis** [comment]
Yes, I'll certainly keep an eye out, or create individual issues. This was more to point out one of several pre-existing standards.

**Comment by kookster** [comment]
Glad you did; I also try to reference similar / related pre-existing specs. RSS has had a ton of adoption, we need to avoid NIH thinking.

**Comment by cio-blubrry** [comment]
We are very open on the RawVoice RSS namespace. anyone can adapt and use it, just as they can media or itunes namespaces.

Just a quick explanation on some of the tags...

<rawvoice:rating> - This is for providing a more specific parental rating utilizing the United States parental rating system. From my research at the time we developed this option, many countries have a way to map these ratings to their own. We (Blubrry) needed this information for TV based applications, Roku, Boxee, Samsung SmartTV, etc... 

<rawvoice:liveEmbed, <rawvoice:flashLiveStream>, <rawvoice:httpLiveStream>, <rawvoice:shoutcastLiveStream>, <rawvoice:liveStream> - Used internally for Blubrry apps. Other developers are welcome to use these tags as well.

<rawvoice:location> - Used as a way for a podcaster to informally indicate in their native language where they podcast from. e.g. Cleveland, Ohio

<rawvoice:frequency> - Used as a way for a podcaster to informatlly indicate in their native language how often they podcast. e.g. Weekly, e.g. Every Monday through Friday

<rawvoice:mycast> - Used internally for Blubrry's subscription service. mycast is a registered trademark of RawVoice, Inc.

<rawvoice:poster> - Specify a specific episode image on a per episode basis. Since we implemented this iTunes now provides the itunes:image at the item level which somewhat duplicates this tag. The image size recommendation is 600 x 600 for this, where-as iTunes has evolved from 300x300, to 600x600 to today's 1400x1400 min and 3000x3000 max. Also important this image can only be a “.jpg” or “.png”, it does not acecpt gif;s, The itunes:image now has the same restriction.

<rawvoice:isHd> - Specifies that the video in the enclosure is in HD format, video that is widescreen (16:9 aspect ratio) with a 720p, 720i, 1080p or 1080i resolution (or better). e.g. It allows for displaying an HD icon in TV Apps such as Roku, Samsung SmartTV, Android TV, etc...

<rawvoice:embed> - Specifies an embed that is associated with the episode. For example, if your episode includes a YouTube video, you can syndicate that embed code along with the episode media file. Blubrry podcast directory will then use that embed in place of the player on the directory.

<rawvoice:webm> - Allows for syndicating an mp4 video (within the enclosure) as well as a webm version as an alternative.

<rawvoice:mp4> - Allows for syndicating a webm video (within the enclsoure) as well as an mp4 version as an alternative.

<rawvoice:metamark> - Allows for syndicating any number of meta information with time positions. Similar to Chapter marks found in m4a, these metamarks have an endless amount of possibilities for tagging ad placement, comments, images, links, audio and video at a specific point in the of the original audio/video.

<rawvoice:subscribe> - Allows for syndicating the subscribe links to podcast directories. iTunes, Google Play, TuneIn, Stitcher and Blubrry are included up to this point. Also import to note that it includes the feed URL as well as an HTML value. The html="URL" allows for providing a landing page to your own subscribe page. For example: html="http://www.powerpresspodcast.com/subscribe-to-podcast/"

Tags <rawvoice:webm>, <rawvoice:mp4> in my opinion have outlived their usefulness, though those passionate about webm's adoption may still encourage it's use.

The <rawvoice:subscribe> tag is used by Blubrry today to display the correct itunes, GooglePlay, Stitcher, TuneIn and Blubrry subscribe options on both blubrry.com and on subscribeonandroid.com.

Please let me know if you have questions.
